### PR TITLE
Remove duplicate URI for IBEG

### PR DESCRIPTION
**Pull request by Riadabd**
## Ticket Number

DL-5770

## PR Description

This PR adds a migration to remove a duplicate organization URI, its properties + reverse properties and its connected bestuursorganen and bestuursorganen in tijd.

### CODE REVIEW

File: config/migrations/2024/20240903164444-remove-duplicate-uri-for-ibeg.sparql [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  GRAPH ?g {
+    ?bestuursorgaanInTijd ?p ?o .
+  }
+}
+WHERE {
+  VALUES ?bestuursorgaanInTijd {
+    <http://data.lblod.info/id/bestuursorganen/7264b6d0-f123-493a-8fe2-4802b5be1aec>
+    <http://data.lblod.info/id/bestuursorganen/8bcdcde8-edeb-4cb5-aae3-0c147d631abc>
+    <http://data.lblod.info/id/bestuursorganen/7ac7d2cb-5fc6-4f36-a6dd-8813cd3e6f3a>
+    <http://data.lblod.info/id/bestuursorganen/b2bea1a43457ea68652f2f7d9b3fa396e952b7fb80d782fcc8660fea42d0c5d5>
```
Comment (mirdono) [inline config/migrations/2024/20240903164444-remove-duplicate-uri-for-ibeg.sparql:36]: **Issue:** This particular governing body also has a function associated with it. It is probably best to remove that as well.

Query used to check for any related functions:
```sparql
PREFIX lblodlg: <http://data.lblod.info/vocabularies/leidinggevenden/>

SELECT DISTINCT ?body ?p ?o
WHERE {
  GRAPH ?g {
    ?body lblodlg:heeftBestuursfunctie ?post .
    ?post ?p ?o .
  }
  VALUES ?body {
    <http://data.lblod.info/id/bestuursorganen/7264b6d0-f123-493a-8fe2-4802b5be1aec>
    <http://data.lblod.info/id/bestuursorganen/8bcdcde8-edeb-4cb5-aae3-0c147d631abc>
    <http://data.lblod.info/id/bestuursorganen/7ac7d2cb-5fc6-4f36-a6dd-8813cd3e6f3a>
    <http://data.lblod.info/id/bestuursorganen/b2bea1a43457ea68652f2f7d9b3fa396e952b7fb80d782fcc8660fea42d0c5d5>
    <http://data.lblod.info/id/bestuursorganen/2a96151e-cbf1-42f6-ac5d-3f9780b3667d>
  }
}
```

Note: I also checked for any mandates (replace `lblodlg:heeftBestuursfunctie` by `org:hasPost` in the above query), but that did not give any results.

**Review by mirdono — CHANGES_REQUESTED** [review]
I left one remark about some missed related data. Other than that it looks fine to me.

File: config/migrations/2024/20240903164444-remove-duplicate-uri-for-ibeg.sparql (hunk 1 above)
Comment (Riadabd) [reply config/migrations/2024/20240903164444-remove-duplicate-uri-for-ibeg.sparql:56]: Nice catch! Added the relevant snippet to delete these properties.

**Review by mirdono — APPROVED** [review]
Looks good to me. Feel free to merge to master and I will include it in the next (pre-) release, which should be released this sprint.

**Comment by Riadabd** [comment]
@mirdono Thanks for the review :)

**Pull request merged**